### ISSUE TITLE
Crosshair Menu looks like a png

### ISSUE THREAD

**Issue by 0w0u** [lacymorrow/crossover#68]
It's not my problem, I can not explain this so well but my friend says that the crosshair menu stills like a png even pressing toggle shortcut 
![image](https://user-images.githubusercontent.com/54691517/115946372-572be700-a486-11eb-8f26-bc2934c3c11c.png) (He can move the cursor, but the menu stills)

**Comment by nipnipj** [comment]
What OS are you guys using? I haven't found any issues with latest release `v1.3.0` in Linux Mint 20.1, not even when using a custom crosshair.

**Comment by lacymorrow** [comment]
Is that Windows 95/ME?

**Comment by ezn17** [comment]
> Is that Windows 95/ME?

No, it's windows 7 home premium, the theme of windows is of windows 95.

**Comment by lacymorrow** [comment]
Sorry for the delay, is this issue only occurring in a specific game?

**Comment by ezn17** [comment]
no, it looks like this in all programs

**Comment by lacymorrow** [comment]
So to be clear, it looks like you are unable to make the white background behind the crosshair disappear?

When you presses the shortcut to lock the crosshair (`ctrl`-`alt`-`shift`-`x`), what happens?

I haven't tested on Windows 7 but I can take a look this weekend.

**Comment by ezn17** [comment]
Sorry for the delay, Yes, i can't make the white background dissapear, also the bottons still there but i can't interact whit them, when i press **ctrl+alt+shift+x** the interface stills like an image, and i can't interact with the buttons until i press **ctrl+alt+shift+x** again.

**Comment by ezn17** [comment]
Sorry for the delay, Yes, i can't make the white background dissapear, when i press ctrl+alt+shift+x the interface stills like an image, and i can't interact with the buttons until i press ctrl+alt+shift+x again.

**Comment by lacymorrow** [comment]
@ezn17 this is a new one, unfortunately you are the first person having transparency issues on windows at the moment, are you using the most recent version of the app? (it looks like it, but I have to ask)

**Comment by ezn17** [comment]
Yes, I have the latest version of crossover and it keeps looking like this.

**Comment by lacymorrow** [comment]
@ezn17 I can only assume it has something to do with the Windows 95 theme you are using, or potentially Windows 7 itself, since I have not personally tested it there. I can only recommend that you:
- Disable the non-standard theme
- Backup and reformat your PC in case there is a software issue
- Upgrade to windows 8/10 (free)

in that order, testing to see if the app works before going onto the next step

**Comment by ezn17** [comment]
Ok, I'll try to do that. Thanks a lot.

**Comment by bob04609** [comment]
I have exactly the same problem in Windows 7 pro 64-bit.  The app background is non-transparent white. Running v2.4.1, just downloaded. Has it been concluded that this in indeed a Windows 7 issue and that the app is in fact incompatible with windows 7? Does anyone have it working with Windows 7?

**Comment by lacymorrow** [comment]
@bob04609 are you using any sort of non-standard theme? I'll try to do some testing in Windows 7 this weekend.

**Comment by bob04609** [comment]
No. Absolutely standard Windows theme included in Windows 7. Nothing fancy or 3rd party.  Currently using "Windows Classic", but it's the same in the "Windows 7 Basic" theme. White, non-transparent, background. Everything else seems to work just fine. Thanks.

**Comment by bob04609** [comment]
I tried it on a Windows 10 machine and it worked fine. I also noticed on the Windows 7 machine that when you resize the cross-hair it leaves the old one in place (but in grey). You can get the old one to fade out by opening and closing the resize menu a few times.

**Comment by ezn17** [comment]
I´m now using windows 7 ultimate, and the problem is solved, now crossover works correctly, i had this problem whit windows 7 home premium. I hope that helps you.

**Comment by lacymorrow** [comment]
Thanks for the digging guys, I'm not sure why but seems like Windows 7 Home just has issues with electron.

**Comment by lacymorrow** [comment]
By the way, do you know if you were running 32-bit or 64-bit?

**Comment by ezn17** [comment]
Windows 7 ultimate 64 bit version.

**Comment by lacymorrow** [comment]
Windows 7 ultimate 64
Snapd: https://github.com/lacymorrow/crossover/issues/213

**Comment by Miko0187** [comment]
I have issue but on Arch Linux

**Comment by begin-theadventure** [comment]
> I have issue but on Arch Linux

@Miko0187 Have you tried the AUR package?

**Comment by Miko0187** [comment]
> > I have issue but on Arch Linux
> 
> @Miko0187 Have you tried the AUR package?

Yes, i have tried the "crossover-overlay-bin" package. Its still not working

**Comment by begin-theadventure** [comment]
Uhh, I just tried the official `.pacman` and I have the same issue, also with  `.rpm`, it doesn't occur with the AUR package.
>404 from electron25 while installing

Try updating mirrors.

**Comment by lacymorrow** [comment]
Weird, I'll try doing that.

**Comment by Miko0187** [comment]
The AUR package works fine now, which fixes the transparency issue for me

**Comment by ShaunaTheDead86** [comment]
I was having the same issue and I was able to get it working on Pop!_OS by downloading version 3.1.5 and running it using the `--no-sandbox` Electron command line option.

**Comment by nipnipj** [comment]
> I was having the same issue and I was able to get it working on Pop!_OS by downloading version 3.1.5 and running it using the `--no-sandbox` Electron command line option.

This also work on Linux mint 21.3

**Comment by lacymorrow** [comment]
📋 Tracked in Paperclip as **LAC-1373**. This issue has been added to our internal project management system and will be reviewed by the engineering team.

**Comment by lacymorrow** [comment]
📋 Tracked in Paperclip as **LAC-1809**. This issue has been added to our internal project management system and will be reviewed by the engineering team.

**Comment by lacymorrow** [comment]
Tracked in Paperclip as **LAC-1373**. This issue has been added to our internal project management system and will be reviewed by the engineering team.

**Comment by lacymorrow** [comment]
Transparency rendering has been reworked multiple times since 2021. If you're seeing transparency problems on the current version, open a new issue with your OS, GPU driver, and a screenshot.
